pom.xml

```diff
@@ -860,7 +860,7 @@
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
         <identity.governance.version>1.11.27</identity.governance.version>
-        <carbon.identity.framework.version>7.8.1</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.8.13-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Let's update this to the proper version

Reply:
addressed with https://github.com/wso2/identity-api-server/pull/773/commits/8c0d3041a47a1a2b301290dae3389ca128bbb20c